src/ordering/ordering_driver.cpp

```
@@ -88,46 +96,52 @@ do_ordering(
-        auto vertices(pgrouting::extract_vertices(edges));
-        UndirectedGraph undigraph(vertices);
-        undigraph.insert_edges(edges);
 
-        if (which == 0) {
-            results = sloanOrdering(undigraph);
-        } else if (which == 2) {
-            results = kingOrdering(undigraph);
+        auto vertices = which == 0 || which == 2?
+            pgrouting::extract_vertices(edges)
+            : std::vector<pgrouting::Basic_vertex>();
+
```

Cppcheck (2.19.0)

[performance] Function parameter 'results' should be passed by const reference.

(passedByValue)